python/tidy/servo_tidy/tidy.py

```diff
@@ -694,6 +664,81 @@ def check_spec(file_name, lines):
                 brace_count -= 1
 
 
+def check_config_file(config_file, print_text=True):
+    # Load configs from servo-tidy.toml
+    if os.path.exists(config_file):
+        with open(config_file) as content:
+            conf_file = content.read()
+            lines = conf_file.splitlines(True)
+    else:
+        print("%s config file is required but was not found" % config_file)
+        sys.exit(1)
```

Comment:
could be just,
``` python
if not os.path.exists(...):
    print("not found blah blah")
    sys.exit(1)
```

... so that we can isolate this check from everything else, and proceed with opening the file.